common/djangoapps/student/helpers.py

```diff
@@ -80,12 +83,18 @@ def check_verify_status_by_course(user, course_enrollments):
         user, queryset=verifications
     )
 
+    # Retrieve expiration_datetime of most recent approved verification
+    # To avoid another database hit, we re-use the queryset we have already retrieved.
+    expiration_datetime = SoftwareSecurePhotoVerification.get_expiration_datetime(user, verifications)
+
     # Retrieve verification deadlines for the enrolled courses
     enrolled_course_keys = [enrollment.course_id for enrollment in course_enrollments]
     course_deadlines = VerificationDeadline.deadlines_for_courses(enrolled_course_keys)
 
     recent_verification_datetime = None
 
+    verification_expiring_soon = SoftwareSecurePhotoVerification.is_verification_expiring_soon(expiration_datetime)
```

Comment:
Put this on line 89. There's no reason to keep it so far from the other line you're adding, which is related.